src/main/java/mate/academy/exception/EntityNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package mate.academy.exception;
+
+public class EntityNotFoundException extends RuntimeException {
```

Comment:
It's a good practice to provide a serialVersionUID for classes that implement Serializable, such as RuntimeException. This helps with version control of the class.